### DISCUSSION TITLE
Add option to reboot containers in LXC update script

### DISCUSSION THREAD

**Discussion by xhemals** [community-scripts/ProxmoxVE#447] [Ideas]
As the name suggests, it would be useful to have an option to reboot the containers after updates as sometimes updating packages can break things without rebooting the machine. And it can become tedious to reboot all machines manually/encounter issues further down the line if the machine hasn't rebooted

**Comment by Mellowlynx** [comment]
Do you have an example of an LXC app that's needing this?
I run a lot of different stuff, and only need to reboot the Linux VM's from time to time.
All my LXC's run fine with no reboot after update.
